public/app/core/components/NavBar/Next/NavBarToggle.tsx

```diff
@@ -16,6 +16,7 @@ export const NavBarToggle = ({ className, isExpanded, onClick }: Props) => {
 
   return (
     <IconButton
+      aria-label={isExpanded ? 'Collapse navigation menu' : 'Expand navigation menu'}
```

Comment:
Maybe "Open navigation" and "Close menu"?

Reply:
@joshhunt i think they should be consistent since they open/close the same thing. so i would vote for either:
- `Open navigation menu`/`Close navigation menu`
- `Open navigation`/`Close navigation`
- `Open menu`/`Close menu`

wdyt? 🤔